selection_sort/__init__.py

```diff
@@ -0,0 +1,20 @@
+def find_smallest(arr):
+    smallest_index = 0
+    smallest = arr[smallest_index]
+
+    for i in range(1, len(arr)):
+        if arr[i] < smallest:
+            smallest_index = i
+            smallest = arr[i]
+
+    return smallest_index
+
+
+def selection_sort(arr):
+    res = []
+
+    for _ in range(len(arr)):
+        smallest = find_smallest(arr)
```

Comment:
`smallest` in this function is not consistent with `smallest` in `find_smallest`, man will be confused. `smallest_index` is better

Reply:
Man moved the function into the bag